builtin/logical/pki/path_intermediate.go

```diff
@@ -196,7 +197,7 @@ func (b *backend) pathSetSignedIntermediate(
 		return nil, err
 	}
 
-	entry.Key = "certs/" + cb.SerialNumber
+	entry.Key = "certs/" + strings.ToLower(strings.Replace(cb.SerialNumber, ":", "-", -1))
```

Comment:
Can we have this creation of `hyphenSerial` in a function and use it in all the places? Seems like we are doing it in a couple of places. Easy to miss out if some if we make changes to this.